### DISCUSSION TITLE
How to sync to the latest version?

### DISCUSSION THREAD

**Discussion by danieljqs1** [FlowiseAI/Flowise#415] [Q&A] [answered]
Hi there, loving Flowise AI!

Quick question coming from a no-code background, how can I update (sync) my flowise to the latest version - I deployed in Render.
Also, will this action erase my current Chatflows?
Is there a best practice on how often to sync?

Thanks a lot!!

**Comment by HenryHengZJ** [comment, marked as answer]
Simply click Sync on your forked repo:
![image](https://github.com/FlowiseAI/Flowise/assets/26460777/15fb3ecf-1160-4986-bfb8-3d4eef8bf7ce)

also flows will be lost if you dont have persistent storage on Render, here's the guide on how to enable that - https://docs.flowiseai.com/deployment/render#persistent-disk

**Reply by danieljqs1** [reply]
Worked great! Thanks a lot!!

**Reply by rinaldo77** [reply]
I did that, but the Flowise version on my machine is still the same?!

**Reply by matthias** [reply]
I assume you have a local git setup on your machine.

Do a `git pull` on your machine.
Then `pnpm install` (just in case any dependencies where updated) and `pnpm build`

**Comment by nsitnov** [comment]
But I am using AWS 
How to update flowise from linux command prompt?
